lessons/Algebra2/quadratic4-other-forms/langs/en-us/pages/build-from-samples-batting-data.adoc

```diff
@@ -0,0 +1,69 @@
+= Build a Model from Samples - Batting Data
+++++
+<style>
+/* Make autonums inside tables look consistent with those outside */
+td .autonum::after { content: ')' !important; }
+
+/* Adjust spacing on the top table */
+.topTable {margin: 1em 0;}
+.topTable td { margin: 0; padding: 0; }
+
+/* Shorten fill in the blanks */
+.studentAnswerShort { width: 5em !important; }
+</style>
+++++
+
+[.linkInstructions]
+We're going to compute a quadratic function from 3 representative points in the `curve-table` we've been working with. While our model would be most accurate if we kept track of all of the decimal values, for today we are going to simplify our calculations by doing some rounding.
+
+[.topTable, cols="12a,1,12a", frame="none", grid="none"]
+|===
+|
+@vspace{1ex}
+@image{../images/curve-angle-distance.png, 400}
+|
+|
+@n Identify three *whole-number* coordinate pairs we can use to summarize a curve that would model the point cloud:
+
+  * a best guess for where the curve *crosses the y-axis* @fitb{}{}
+
+  * a point *from the middle* of the curve @fitb{}{}
+
+  * a point *from the right side* of the curve @fitb{}{}
+
+@vspace{1ex}
+
+@n Fill in the x and y-values in the *standard form* models below using the three points you found:
+
+@indented{
+@fitbruby{6em}{}{y (mpg)} = A(@fitbruby{6em}{}{x (speed)})^2^ + B(@fitbruby{6em}{}{x (speed)}) + C
+
+@fitbruby{6em}{}{y (mpg)} = A(@fitbruby{6em}{}{x (speed)})^2^ + B(@fitbruby{6em}{}{x (speed)}) + C
+
+@fitbruby{6em}{}{y (mpg)} = A(@fitbruby{6em}{}{x (speed)})^2^ + B(@fitbruby{6em}{}{x (speed)}) + C
+}
+|===
+
+@n In the space below - or on another sheet of paper - solve this series of equations for A, B, and C. +
+_As we solve the series, we'll likely get some very long decimal values. You can round these values to the ten thousandths place as we work._
```

Comment:
I would even go so far as to drop this to the thousandths place (or even hundredths?)